cmd/shell.go

```diff
@@ -288,23 +288,23 @@ func newStatsRegistry(addr string, logger blog.Logger) prometheus.Registerer {
 	}))
 
 	if addr == "" {
-		logger.Err("Debug listen address is not configured")
-		os.Exit(1)
-	}
-	logger.Infof("Debug server listening on %s", addr)
+		logger.Infof("No debug server specified")
```

Comment:
```suggestion
		logger.Infof("No debug listen address specified")
```

The old "listen address" terminology is a little more correct. "No debug server specified" sounds like Boulder was expecting a remote server to talk to as opposed to a local address to listen on.

Also, even though we are getting rid the `Exit(1)`, this is a good place to make use of the [early return pattern](https://go.dev/doc/effective_go#if): after the log statement, `return registry()`. Then the `else` can be removed and the following lines outdented.

Speaking of which, there's another refactoring opportunity at hand: We could move this check all the way up to the top of the function, immediately after `prometheus.NewRegistry()`. That makes it clear that when there's no debug port, the HTTP server is not even created.